src/Domain/GetSongsList.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+function getSongs()
+{
+    $dataList=file_get_contents('../../repository/songs_list.json');
+    $listOfSongs= json_decode( $dataList , true);

```

Comment:
Same as above, regarding a space before the equals sign. Remove the space after the opening parenthesis and before the comma in the argument list (https://www.php-fig.org/psr/psr-12/#47-method-and-function-calls).